translations/messages-en.properties

```diff
@@ -978,4 +978,5 @@ instance.upgrade.confirm = Confirm upgrade
 instance.upgrade.confirm.warning.text = This action cannot be undone, make sure your data has been backed up\!
 confirm.delete.progress = Deletion in progress
 confirm.delete.totals = {{totalDeleted}}/{{totalSelected}} deleted...
-messages.unknown.sender = Unknown sender
\ No newline at end of file
+messages.unknown.sender = Unknown sender
+report.sent.by = Sent by
```

Comment:
Use a parameterised translation, eg:

```
report.sent.by = Sent by {{name}}
```

This makes the translation much more flexible and means it'll work with languages where the name is written first. It will make your template a little more complicate though...

Reply:
Done.